providers/catalog.go

```diff
@@ -2148,4 +2149,27 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Insightly API Key authentication
+	Insightly: {
+		AuthType: ApiKey,
+		BaseURL:  "https://api.na1.insightly.com",
```

Comment:
<img width="599" alt="Screenshot 2024-06-19 at 12 07 26 PM" src="https://github.com/amp-labs/connectors/assets/18614743/01b76938-21b2-4042-85a8-a69f65840e39">
@laurenzlong `na1` is the `pod` here (basically a region), and we can kind of get away with substituting it with `workspace` here. This is information that cannot be fetched via an API call. What do you think?

Reply:
@laurenzlong Descoping this for now